### DISCUSSION TITLE
Issue creating profile for SFTP

### DISCUSSION THREAD

**Discussion by scottharman** [iterate-ch/cyberduck#14588] [Q&A]
Trying to generate a new profile to allow a user to quickly transfer some files, but even with a minimal config, the username and password fields aren't populated correctly, and the display names are overwritten

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Protocol</key>
	<string>sftp</string>

	<key>Description</key>
	<string>Test Secure Server Profile</string>
        <key>Default Hostname</key>
        <string>test.com</string>
        <key>Default Port</key>
        <string>22</string>
		<key>Username Placeholder</key>
        <string>should_populate_username_field</string>
        <key>Password Placeholder</key>
        <string>should_be_password</string>
</dict>
</plist>
``` 
<img width="366" alt="image" src="https://user-images.githubusercontent.com/671934/235014704-66f642c3-89c5-4426-9bbd-b9919b4b1fd8.png">

**Comment by AliveDevil** [comment]
There are two different things here - a Connection Profile (`.cyberduckprofile`) and a Cyberduck Bookmark-file (`.duck`).
A connection profile doesn't provide any credential information, and the X placeholder is only used for labeling purposes.

You can create a bookmark in Cyberduck with your configuration, then drag that out of the Window on to a filesystem and you have a distributable bookmark users just have to double click to connect to. Both a Connection Profile and a Cyberduck Bookmark won't include any passwords.

**Reply by scottharman** [reply]
According to the docs, Username Placeholder and Password Placeholder appear to be valid - is this not the case?

https://docs.cyberduck.io/protocols/profiles/

Either way, declaring the value shouldn't override the username and password field names and break the connection profile.

**Reply by AliveDevil** [reply]
One thing I see in the attached profile is the missing "<key>Vendor</key>", that must be unique (i.e. "test-secure"). The behavior of Cyberduck in this case might be unstable.

Otherwise, we'd need to update the docs, to specify that "Username Placeholder" and "Password Placeholder" are only used for the respective input fields label - they are not meant to provide any default for usernames/passwords.

**Reply by dkocher** [reply]
On macOS, username and password placeholders are in addition to the label displayed in the input field as a default placeholder text.
